BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEvent.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api
+
+import androidx.room.PrimaryKey
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+
+// NEXT MAJOR VERSION: Convert to data class, we're unable to do so now because the
```

Comment:
[Data classes in Kotlin](https://kotlinlang.org/docs/data-classes.html) seem super neat. I don't believe Swift has anything comparable, so was cool to read about.

Reply:
Yeah I'd say `struct` is close. It's a good cross-platform analogy for the most part.